apps/src/amazonFutureEngineerEligibility/amazonFutureEngineerEligibility.js

```diff
@@ -0,0 +1,22 @@
+import $ from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import AmazonFutureEngineerEligibility from '@cdo/apps/templates/amazonFutureEngineerEligibility';
+
+$(document).ready(init);
+
+function showAmazonFutureEngineerEligibility() {
+  const amazonFutureEngineerEligibilityElement = $(
+    '.amazon-future-engineer-eligibility-container'
+  );
+
+  ReactDOM.render(
+    // Need to update API endpoint and source page ID (not sure if even needed)
```

Comment:
Does "source page ID" refer to the IDS we pass in the Regional Partner Mini-contact form?  We did that so that we could track which pages were generating leads.

Reply:
Yes, sorry I missed removing this comment when I removed the `sourcePageId` prop. Thanks for the context, was wondering why that prop was there.